webapp/apps/dynamic/models.py

```diff
@@ -87,6 +87,27 @@ class DynamicBehaviorSaveInputs(models.Model):
     micro_sim = models.ForeignKey(OutputUrl, blank=True, null=True,
                                   on_delete=models.SET_NULL)
 
+    def get_tax_result(self):
+        """
+        If taxcalc version is greater than or equal to 0.13.0, return table
+        If taxcalc version is less than 0.13.0, then rename keys to new names
+        and then return table
+        """
+        outputurl = OutputUrl.objects.get(unique_inputs__pk=self.pk)
+        taxcalc_vers = outputurl.taxcalc_vers
+        # only the older (pre 0.13.0) taxcalc versions are null
+        if taxcalc_vers:
+            taxcalc_vers = LooseVersion(taxcalc_vers)
```

Comment:
I feel like this should break something because LooseVersion isn't imported... Maybe I'm missing something

Reply:
Nope, thanks for catching that. I added the import statement and a test for testing this method on the dynamic behavioral side.